### PR TITLE
glide type feature added

### PR DESCRIPTION
**Pull request by surajsahani**
This pull request involves renaming and updating package references across multiple files to ensure consistency in the project structure. The most important changes include renaming the packages from `org.inputmethod` to `com.android.inputmethod` and updating the import statements accordingly.

### Package renaming and import updates:

* [`app/src/main/java/com/android/inputmethod/keyboard/ProximityInfo.java`](diffhunk://#diff-dad7c7cfef8b705f997e35a637bc825f4d5c77fa7743d974d9f0c9eb1ff54d39L1-R1): Renamed package from `org.inputmethod.keyboard` to `com.android.inputmethod.keyboard`.
* [`app/src/main/java/com/android/inputmethod/latin/BinaryDictionary.java`](diffhunk://#diff-857636beccc302df3a69d4e83b97577561a41b5f6a58dd5412b74a85256d36fbL1-R10): Renamed package from `org.inputmethod.latin` to `com.android.inputmethod.latin` and updated import statements for `BinaryDictionaryUtils` and `WordInputEventForPersonalization`. [[1]](diffhunk://#diff-857636beccc302df3a69d4e83b97577561a41b5f6a58dd5412b74a85256d36fbL1-R10) [[2]](diffhunk://#diff-857636beccc302df3a69d4e83b97577561a41b5f6a58dd5412b74a85256d36fbL22-L23)
* [`app/src/main/java/com/android/inputmethod/latin/DicTraverseSession.java`](diffhunk://#diff-f7385542632f92ebca7bd1719b23a8195a8fc59d8d523dfc8c9894e857e422dbL1-R1): Renamed package from `org.inputmethod.latin` to `com.android.inputmethod.latin`.
* [`app/src/main/java/com/android/inputmethod/latin/utils/BinaryDictionaryUtils.java`](diffhunk://#diff-3f41b3ec2969d7671c79b80400ee03c93637d33d1c766e3afa2cdb11bcb8f373L1-L3): Renamed package from `org.inputmethod.latin.utils` to `com.android.inputmethod.latin.utils` and updated import statement for `BinaryDictionary`.
* [`app/src/main/java/com/android/inputmethod/latin/utils/WordInputEventForPersonalization.java`](diffhunk://#diff-d91580d09b562c35764d7f837bf215f79f0e701a249d9addb74df3d4ba9df9aeL1-R1): Renamed package from `org.inputmethod.latin.utils` to `com.android.inputmethod.latin.utils`.

### Additional updates to import statements:

* [`app/src/main/java/org/oscar/kb/keyboard/Keyboard.java`](diffhunk://#diff-ae39dccdf138bd7e795287e5ad283a76118367b89a167f43a8d3851d553dad9eL14-R15): Updated import statement for `ProximityInfo` to reflect the new package name.
* [`app/src/main/java/org/oscar/kb/keyboard/KeyboardLayout.java`](diffhunk://#diff-65c1621d0459ab7869a43f12acf44a1fd9a4f23b1deabe8e5f977736674957ecL11-R12): Updated import statement for `ProximityInfo` to reflect the new package name.
* [`app/src/main/java/org/oscar/kb/latin/ContactsBinaryDictionary.java`](diffhunk://#diff-cd3089a2054b6f1aa37e030b05e7300c8bee826fe14b89dab41a766fd007bbccL18-R19): Updated import statement for `BinaryDictionary` to reflect the new package name.
* [`app/src/main/java/org/oscar/kb/latin/ExpandableBinaryDictionary.java`](diffhunk://#diff-d25ddf579c2fc96433f88cb9edc5ffbe1af6392d8c794fbe7bb65ea6533d8dbfL14-R15): Updated import statement for `BinaryDictionary` to reflect the new package name.
* [`app/src/main/java/org/oscar/kb/latin/ReadOnlyBinaryDictionary.java`](diffhunk://#diff-7c58e093e7d6ca74da0763310eb96a1d39af6d30ee96b6638b8afec6873333e0L9-R10): Updated import statement for `BinaryDictionary` to reflect the new package name.

### Minor code corrections:

* [`app/src/main/java/org/oscar/kb/latin/setup/SetupWizardActivity.java`](diffhunk://#diff-84d49020eb0cc858907a72e08febfb423e50c4cd191848069ff47a5444681344L149-R150): Fixed a syntax error by removing an unnecessary semicolon.See the contributing readme for more detailed guideline, please understand and accept them: https://github.com/Helium314/HeliBoard/blob/main/CONTRIBUTING.md
tl;dr (you should still read the full list though):
* make sure it's wanted
* a single thing only
* describe it properly
* re-use existing mechanisms / code
* low performance impact
* make it a draft if you still want to work on it
* no translations or dictionaries

Further
* When the PR contains "fixes" <issue number>, the related issue will be linked and automatically closed if the PR is merged (also works for other words like "fix", "resolve", "resolves", "closes", ...)
* If you add a keyboard layout, make sure you also read https://github.com/Helium314/HeliBoard/blob/main/layouts.md#adding-new-layouts--languages
* Please avoid force-pushing when doing changes. This way it's not possible to see which parts have changed since the previous state.

### CODE REVIEW

**Comment by surajsahani** [comment]
completed #59
